tests/e2e/machine_pool_test.go

```diff
@@ -353,6 +353,78 @@ var _ = Describe("TF Test", func() {
 
 			})
 		})
+		Context("Author:amalykhi-High-OCP-65063 @OCP-65063 @amalykhi", func() {
+			It("Author:amalykhi-High-OCP-65063 Create single-az machinepool for multi-az cluster", ci.Day2, ci.High, ci.FeatureMachinepool, func() {
+				if !profile.MultiAZ {
+					Skip("The test is configured for MultiAZ cluster only")
+				}
+				getResp, err := cms.RetrieveClusterDetail(ci.RHCSConnection, clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				azs := getResp.Body().Nodes().AvailabilityZones()
+				By("Create additional machinepool with availability zone specified")
+				replicas := 3
```

Comment:
Better to use replicas 1 or 2 for the testing which will cover the replicas validation well.

Reply:
fixed